noir-projects/aztec-nr/aztec/src/context/private_context.nr

```diff
@@ -496,7 +496,7 @@ impl PrivateContext {
         let mut is_static_call = is_static_call | self.inputs.call_context.is_static_call;
         /// Safety: TODO(https://github.com/AztecProtocol/aztec-packages/issues/8985): Fix this.
         /// WARNING: This is insecure and should be temporary!
-        /// The oracle repacks the arguments and returns a new args_hash.
+        /// The oracle hashes the arguments and returns a new args_hash.
```

Comment:
The mental model of packing into a hash is now illegal and frowned upon. Forgot to fix that in a PR down the stack before it got merged.